.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+name: Stable release 🕊️📦
+
+on:
+  release:
+    types:
+      - released
+      - prereleased
+
+jobs:
+  docker:
+    name: Build Docker images 💿💻
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout ⬇️
+        uses: actions/checkout@v2.4.0
+
+      - name: Setup node environment ⚙️
+        uses: actions/setup-node@v2.5.0
+        with:
+          node-version: 16
+          cache: 'npm'
+          check-latest: true
+
+      - name: Install dependencies 📦
+        run: npm ci --no-audit
+
+      - name: Build client 🛠️
+        run: npm run build
+
+      - name: Configure QEMU ⚙️
+        uses: docker/setup-qemu-action@v1.2.0
+
+      - name: Configure Docker Buildx ⚙️
+        uses: docker/setup-buildx-action@v1.5.1
+
+      - name: Login to Docker Hub 🔑
+        uses: docker/login-action@v1.10.0
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Login to GitHub Container Registry 🔑
+        uses: docker/login-action@v1.10.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.JF_BOT_TOKEN }}
+
+      - name: Build and push stable images 🛠️⬆️
+        if: github.event.action == 'released'
+        uses: docker/build-push-action@v2.6.1
+        with:
+          context: .
+          push: true
+          platforms: linux/amd64,linux/arm64,linux/arm,linux/arm/v6
+          file: '.github/workflows/Dockerfile.nginx'
+          tags: |
+            jellyfin/jellyfin-vue:stable
+            jellyfin/jellyfin-vue:stable.${{ github.event.release.tag_name }}
+            jellyfin/jellyfin-vue:latest
+            ghcr.io/jellyfin/jellyfin-vue:stable
+            ghcr.io/jellyfin/jellyfin-vue:stable.${{ github.event.release.tag_name }}
+            ghcr.io/jellyfin/jellyfin-vue:latest
+
+      - name: Build and push release candidate images 🛠️⬆️
+        if: github.event.action == 'prereleased'
+        uses: docker/build-push-action@v2.6.1
+        with:
+          context: .
+          push: true
+          platforms: linux/amd64,linux/arm64,linux/arm,linux/arm/v6
+          file: '.github/workflows/Dockerfile.nginx'
```

Comment:
Since we only have one Dockerfile now maybe we can drop the suffix at the end?

Reply:
@dkanada Renamed to ``Dockerfile.ci``, so it's clear the distinction between the "normal" Dockerfile (where the client is also built in the client context) and this one, where the client is first built in CI and, after that, copied to a Docker image.

Reply:
I wonder why those need to be different in the first place but this is a good change so it sounds fine to me.

Reply:
@dkanada The main one builds the client entirely in the Docker context while the CI one takes the building output from the previous step (which performs the build in the host of the CI environment), saving resources and time.

Reply:
The time savings might be useful, but I honestly couldn't say without specific values for comparison. Trying to improve runtime by sidestepping the official image sounds entirely counterproductive though - the whole point of CI is to test our build, but in this case we're only testing the native build, not the Docker image that gets shipped to users.